pkg/util/tochar/cache_test.go

```diff
@@ -40,3 +40,17 @@ func TestFormatCache(t *testing.T) {
 	_, ok = c.mu.cache.Get("HH24")
 	require.True(t, ok)
 }
+
+// TestFormatCacheLookupThreadSafe tests that FormatCache.lookup is thread safe.
+// This test is non-deterministic and flakes indicate it is not thread safe.
```

Comment:
nit: i think the comment should be updated. after your fix, it should not be flaky right?

Reply:
The comment isn't saying that the test is flaky, but if it flakes then the code being tested is probably not thread safe.

Reply:
ah ok. i read at as "This test is non-deterministic and flakes, ..."

maybe add a comma: "This test is non-deterministic, and flakes indicate...

Reply:
Fixed